projects/canopy/src/lib/carousel/auto-play/auto-play.component.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LgAutoplayComponent } from './auto-play.component';
+
+describe('LgButtonComponent', () => {
+  let component: LgAutoplayComponent;
+  let fixture: ComponentFixture<LgAutoplayComponent>;
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(LgAutoplayComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Just wondering if we should add some tests here to cover the visibility of the buttons, based on the condition: `*ngIf="pause.value"` for example.

Perhaps we can also have a test to ensure the `pause` observable is changed on click?